sql/functions.sql

```diff
@@ -236,6 +256,99 @@ END;
 $$
 LANGUAGE plpgsql IMMUTABLE;
 
+
+CREATE OR REPLACE FUNCTION get_postcode_rank(country_code VARCHAR(2), postcode TEXT,
+                                      OUT rank_search SMALLINT, OUT rank_address SMALLINT)
+AS $$
+DECLARE
+  part TEXT;
+BEGIN
+    rank_search := 30;
+    rank_address := 30;
+    postcode := upper(postcode);
+
+    IF country_code = 'gb' THEN
```

Comment:
Could you comment (or put a link to a wiki page) explaining the objective of this function, so that non-nominatim expert could contribute with their country?
(is there anything needed for french postalcodes?)